src/Command/Push/PushDatabaseCommand.php

```diff
@@ -98,7 +98,7 @@ private function uploadDatabaseDump(
     private function importDatabaseDumpOnRemote(EnvironmentResponse $environment, string $remoteDumpFilepath, DatabaseResponse $database): void
     {
         $this->logger->debug("Importing $remoteDumpFilepath to MySQL on remote machine");
-        $command = "pv $remoteDumpFilepath --bytes --rate | gunzip | MYSQL_PWD=$database->password mysql --host={$this->getHostFromDatabaseResponse($environment, $database)} --user=$database->user_name {$this->getNameFromDatabaseResponse($database)}";
+        $command = "pv $remoteDumpFilepath --bytes --rate | gunzip | MYSQL_PWD=" . escapeshellarg($database->password) . " mysql --host={$this->getHostFromDatabaseResponse($environment, $database)} --user=$database->user_name {$this->getNameFromDatabaseResponse($database)}";
```

Comment:
Please convert this to the new-style call.

Please also add `escapeshellarg` as a forbidden function in `Generic.PHP.ForbiddenFunctions` in `phpcs.xml.dist`, and suggest `$localMachineHelper->executeFromCmd` as an alternative.